veg2hab/validation.py

```diff
@@ -343,12 +343,7 @@ def bereken_percentage_confusion_matrix(
             if pred_percentage == 0:
                 break
 
-    # TODO add some validation here!!
-    # willen we hier nog valideren met percentages?
-    #     if percentage > 1e-10:
-    #         logging.warning("Non matching percentages in conf matrix, too much pred %?")
-    # if true_percentage > 1e-10:
-    #     logging.warning("Non matching percentages in conf matrix, too much true %?")
+    # TODO: Some extra validation here would be nice, with warnings if pred/true percentages do not match
```

Comment:
Deze todo laten we staan? Is ook de validatie, dus is niet zo heel belangrijk.

Reply:
Het toevoegen was complexer dan ik hoopte, omdat niet in alle gevallen de loops direct er boven worden uitgevoerd, dus dat heb ik niet gedaan, maar aangezien het in theorie idd best nice is om nog wat validation te hebben heb ik hem wel laten staan.